docs/getting-started/documentation/oauth-data-address.md

```diff
@@ -0,0 +1,125 @@
+How Can OAuth2 be used for Http-Data-Sources and Sinks?
```

Comment:
OAuth2 has a large amount of flows. We should specifiy that we are documenting using the OAuth2 Client Credentials flow.